skaffold.yaml

```diff
@@ -1092,6 +1092,8 @@ deploy:
       setValueTemplates:
         image.repository: '{{.IMAGE_REPO_eu_gcr_io_gardener_project_gardener_gardenlet}}'
         image.tag: '{{.IMAGE_TAG_eu_gcr_io_gardener_project_gardener_gardenlet}}@{{.IMAGE_DIGEST_eu_gcr_io_gardener_project_gardener_gardenlet}}'
+        global.resourceManager.image.repository: '{{.IMAGE_REPO_eu_gcr_io_gardener_project_gardener_resource_manager}}'
+        global.resourceManager.image.tag: '{{.IMAGE_TAG_eu_gcr_io_gardener_project_gardener_resource_manager}}@{{.IMAGE_DIGEST_eu_gcr_io_gardener_project_gardener_resource_manager}}'
```

Comment:
I'm not sure what this is supposed to do. There is no `global.resourceManager.image.repository` tag in gardenlet helm chart and the image of `gardener-resource-manager` is already overwritten using `imageVectorOverwrite` 4 lines below.

Reply:
Artifact of something else I broke, thanks for catching that!